cli/azd/.golangci.yaml

```diff
@@ -2,8 +2,12 @@ linters:
   enable:
     - errorlint
     - gofmt
+    - lll
```

Comment:
Do we get auto formatting on save?

Reply:
No, we only get a yellow highlight like:
<img width="776" alt="image" src="https://user-images.githubusercontent.com/24213737/195404349-35e75928-0cfa-48c9-8f54-fc9931f35895.png">

We can either adjust it or use `//nolint:lll` to disable a code block (like long urls or text)

Reply:
I can bring some suggested extension for VSCode on a following PR if folks want to have on-save formatting to handle this.

But, I would just expect that after this PR, folks would start taking care of line limits naturally